### PR TITLE
Add logbook note

### PR DESCRIPTION
**Pull request by balloob**
## Proposed change

Add a note whenever we show logbook entries in a trace that not all shown logbook entries might be related.

image

image

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/automation/trace/ha-automation-trace-logbook.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      <hat-logbook-note></hat-logbook-note>
+    `;
+  }
+
+  static get styles(): CSSResult[] {
+    return [
+      css`
+        :host {
+          display: block;
+          padding: 16px;
```
Comment (bramkragten) [inline src/panels/config/automation/trace/ha-automation-trace-logbook.ts:36]: This means we now have extra padding on the logbook entries that already have padding?

File: src/panels/config/automation/trace/ha-automation-trace-logbook.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  @property({ attribute: false }) public logbookEntries!: LogbookEntry[];
+
+  protected render(): TemplateResult {
+    return html`
+      <ha-logbook
+        .hass=${this.hass}
+        .entries=${this.logbookEntries}
+      ></ha-logbook>
+      <hat-logbook-note></hat-logbook-note>
```
Comment (bramkragten) [inline src/panels/config/automation/trace/ha-automation-trace-logbook.ts:27]:
```suggestion
    return logbookEntries.length
      ? html`
          <ha-logbook
            relative-time
            .hass=${this.hass}
            .entries=${logbookEntries}
            .narrow=${this.narrow}
          ></ha-logbook>
          <hat-logbook-note></hat-logbook-note>
        `
      : html`<div class="padded-box">
          No Logbook entries found for this step.
        </div>`;
  }
```

File: src/panels/config/automation/trace/ha-automation-trace-logbook.ts (hunk 1 above)
Comment (bramkragten) [inline src/panels/config/automation/trace/ha-automation-trace-logbook.ts:36]:
```suggestion
        .padded-box {
          padding: 16px;
```

Comment (balloob) [reply src/panels/config/automation/trace/ha-automation-trace-logbook.ts:36]: oh this should have been on the note. oops.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**